spec/order_spec.rb

```diff
@@ -0,0 +1,27 @@
+require './lib/order.rb'
+describe Order do
+  describe '#add_items' do
+    it 'allows to enter the dish name, quantity and total amount' do
+      expect{ subject.add_items("Pad_thai", 2, 16) }.to change { subject.my_order }.to ["Pad_thai * 2"]
+    end
+
+    it 'presents the amount to be paid for each dish' do
+      expect{ subject.add_items("Pad_thai", 3, 24) }.to change { subject.my_bill }.to [24]
+    end
+  end
+  describe '#total_amount' do
+    it 'shows the total amount for all dishes in the order' do
+      order = Order.new
+      order.add_items("Pad_thai", 2, 16)
```

Comment:
You can use 'subject' instead of Order.new. Saves you having to define a new variable for each test.